shell/server/pre-meteor.js

```diff
@@ -210,6 +210,25 @@ function checkMagic(buf, magic) {
   return true;
 }
 
+function serveSelfTest(req, res) {
+  inMeteor(() => {
```

Comment:
This inMeteor() is unnecessary since you aren't using any meteor APIs in the code within.